static/app/views/issueList/groupSearchViewTabs/draggableTabBar.tsx

```diff
@@ -1,359 +0,0 @@
-
-export function DraggableTabBar({initialTabKey, router}: DraggableTabBarProps) {
-  // TODO: Extract this to a separate component encompassing Tab.Item in the future
-  const [editingTabKey, setEditingTabKey] = useState<string | null>(null);
-
-  const organization = useOrganization();
-  const location = useLocation();
-
-  const {cursor: _cursor, page: _page, ...queryParams} = router?.location?.query ?? {};
-  const {viewId} = queryParams;
-
-  const {setNewViewActive, setOnNewViewsSaved} = useContext(NewTabContext);
-  const {tabListState, state, dispatch} = useContext(IssueViewsContext);
-  const {views: tabs, tempView: tempTab} = state;
-
-  useHotkeys(
-    [
-      {
-        match: ['command+s', 'ctrl+s'],
-        includeInputs: true,
-        callback: () => {
-          if (tabs.find(tab => tab.key === tabListState?.selectedKey)?.unsavedChanges) {
-            dispatch({type: 'SAVE_CHANGES', syncViews: true});
-            addSuccessMessage(t('Changes saved to view'));
-          }
-        },
-      },
-    ],
-    [dispatch, tabListState?.selectedKey, tabs]
-  );
-
-  const handleNewViewsSaved: NewTabContext['onNewViewsSaved'] = useCallback<
-    NewTabContext['onNewViewsSaved']
-  >(
-    () => (newViews: NewView[]) => {
-      if (newViews.length === 0) {
-        return;
-      }
-      setNewViewActive(false);
-      const {label, query, saveQueryToView} = newViews[0];
-      const remainingNewViews: IssueView[] = newViews.slice(1)?.map(view => {
-        const newId = generateTempViewId();
-        const viewToTab: IssueView = {
-          id: newId,
-          key: newId,
-          label: view.label,
-          query: view.query,
-          querySort: IssueSortOptions.DATE,
-          unsavedChanges: view.saveQueryToView
-            ? undefined
-            : [view.query, IssueSortOptions.DATE],
-          isCommitted: true,
-        };
-        return viewToTab;
-      });
-      let updatedTabs: IssueView[] = tabs.map(tab => {
-        if (tab.key === viewId) {
-          return {
-            ...tab,
-            label,
-            query: saveQueryToView ? query : '',
-            querySort: IssueSortOptions.DATE,
-            unsavedChanges: saveQueryToView ? undefined : [query, IssueSortOptions.DATE],
-            isCommitted: true,
-          };
-        }
-        return tab;
-      });
-
-      if (remainingNewViews.length > 0) {
-        updatedTabs = [...updatedTabs, ...remainingNewViews];
-      }
-
-      dispatch({
-        type: 'SET_VIEWS',
-        views: updatedTabs,
-        syncViews: true,
-        updateQueryParams: {
-          newQueryParams: {query, sort: IssueSortOptions.DATE},
-          replace: true,
-        },
-      });
-    },
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [location, setNewViewActive, tabs, viewId]
-  );
-
-  useEffect(() => {
-    setOnNewViewsSaved(handleNewViewsSaved);
```

Comment:
Moved to `IssueViews.tsx` (context component)

This logic makes sure that the "save as view" and "save all" buttons on the new view page work properly.